src/nxdoc/getting-started/best-practices/nuxeo-platform-feature-table.md

```diff
@@ -474,7 +458,7 @@ The tables below use the following notation:
     <td>N/A</td>
     <td>&#10003;</td>
     <td>&#10003;</td>
-    <td colspan="1">Box, Dropbox, Google Drive, Microsoft OneDrive<br/>
```

Comment:
Wasn't aware that we had drop dropbox and onedrive.

Reply:
We used to, but these two don't work anymore (we were using an old api that has been removed)